src/reference/sectionC_interpretation.inc

```diff
@@ -333,8 +333,7 @@ Interpretation of ``math`` elements
 Interpretation of ``encapsulation`` elements
 --------------------------------------------
 
-1. For the purposes of this specification, there SHALL be a conceptual "encapsulation digraph" in which there is EXACTLY one node for every component in the :ref:`CellML model<specA_cellml_model>`.
-   Therefore the encapsulation digraph will not contain any loops.
```

Comment:
Do we still say somewhere that it can't have loops? I know it's a consequence of the other points, but "for the avoidance of doubt" yada yada?

Reply:
perhaps in (4) below?

Reply:
If you like. You can add it after point 2, cause it follows from 1&2+the rules in chapter 2

But to me it distracted a bit, as the rest of this section is about defining the encapsulated, sibling, and hidden sets

Reply:
Also, if you follow the rules in chapter 2, you have a structure that won't lead to cycles.
So it's already sorted at this point. It's not a rule you need to realise is implied by these points and then check. If you've done chapter 2 right it just can't happen at this point.

Reply:
Could be mentioned in informative spec instead maybe?

Reply:
Having already ruled out loops there seems to be no need to add a reminder statement.  This would also be inconsistent with the rest of the specification.